### PR TITLE
doc/cpufreq: Add docs for max/min frequency methods.

### PR DESCRIPTION
**Pull request by marcbonnici**
Adds documentation for the previously added methods in commit
a1e58cd8b1b293d5a18f204effbe1f37efcbecd4

### CODE REVIEW

File: doc/modules.rst [hunk 1]
```diff
@@ -101,6 +101,12 @@ policies (governors). The ``devlib`` module exposes the following interface
    :param cpu: The cpu; could be a numeric or the corresponding string (e.g.
        ``1`` or ``"cpu1"``).
 
+.. method:: target.cpufreq.get_min_available_frequency(cpu)
+            target.cpufreq.get_max_available_frequency(cpu)
+
+    Retrieve the the minimum or maximum DVFS frequency available for the
+    specified CPU. Returns an int or None if could not be determined.
```
Comment (setrofim) [inline doc/modules.rst]: This should probably go below ``get_<min|max>_frequency`` and the description should clarify how these are different.

Comment (marcbonnici) [reply doc/modules.rst]: Updated.

File: doc/modules.rst [hunk 2]
```diff
@@ -106,11 +106,19 @@ policies (governors). The ``devlib`` module exposes the following interface
...
+   CPU. "set" functions are available with all governors other than
+   ``userspace``.
 
    :param cpu: The cpu; could be a numeric or the corresponding string (e.g.
        ``1`` or ``"cpu1"``).
+
+.. method:: target.cpufreq.get_min_available_frequency(cpu)
+            target.cpufreq.get_max_available_frequency(cpu)
+
+    Retrieve the min or max DVFS frequency that is supported for a given
```
Comment (setrofim) [inline doc/modules.rst]: Might be useful to make the distinction from the above a bit clearer

> Retrieve the min or max DVFS frequency that is supported (as opposed to currently enforced) for a given CPU.

Comment (marcbonnici) [reply doc/modules.rst]: Updated again.

**Pull request merged**